internal/business/persisted_operations/gcp_storage_loader.go

```diff
@@ -33,14 +35,20 @@ func NewGcpStorageLoader(ctx context.Context, bucket string, store string) (*Gcp
 		store:  store,
 	}, nil
 }
-func (g *GcpStorageLoader) Load(ctx context.Context) error {
+func (g *GcpStorageLoader) Load(ctx context.Context, log *slog.Logger) error {
```

Comment:
Instead of passing Logger here, could you add it to the NewGcpStorageLoader struct and access it via `g.log`?